collections/_article/dev-snapshot-godot-4-1-beta-1.md

```diff
@@ -0,0 +1,120 @@
+
+Another core design aspect of Godot is its coordinate system. The engine uses -Z for the forward direction within its 3D environment, and this applies to both camera and models imported into the engine. By convention, 3D assets are usually created facing the camera, which means that they would appear rotated upon being imported into Godot. This introduces a big usability problem for anyone making 3D games in the engine, as some amount of boilerplate is required to use models in levels. After a long and nuanced discussion, Juan, [Tokage](https://github.com/TokageItLab), and [Aaron Franke](https://github.com/aaronfranke) devised a solution that streamlines things without shattering the core of the engine:
+
+  - Add an option to use model space with the `look_at` method and other similar methods ([GH-76082](https://github.com/godotengine/godot/pull/76082)).
+  - Switch "front" and "back" camera views in the editor to be consistent with itself ([GH-76052](https://github.com/godotengine/godot/pull/76052)).
+  - Fix a long-standing `PathFollow` issue with the forward direction ([GH-72842](https://github.com/godotengine/godot/pull/72842)).
+
+Finally, Godot 4.1 brings back frame delta smoothing, first introduced in Godot 3.4 ([GH-52314](https://github.com/godotengine/godot/pull/52314)) by [lawnjelly](https://github.com/lawnjelly). You can read more about it in our [3.4 release blog post](/article/godot-3-4-is-released).
+
+### C#
+
+The team of .NET contributors have bridged the gap between C# and GDScript in Godot 4 in terms of feature parity. However, one glaring missing feature was global classes. In GDScript you can register any class as globally accessible with the `class_name` keyword, which, among other things, makes it appear in various parts of the editor UI. For example, you can create custom resources and instantiate them from the appropriately typed fields in the inspector dock. C# has lacked this functionality, but thanks to efforts from [Raul Santos](https://github.com/raulsntos) and [Will Nations](https://github.com/willnationsdev) (as well as rigorous testing from many other contributors) this is no longer the case ([GH-72619](https://github.com/godotengine/godot/pull/72619)).
```

Comment:
Would it be possible to add a link to the new documentation page about the global classes feature in C#? https://docs.godotengine.org/en/latest/tutorials/scripting/c_sharp/c_sharp_global_classes.html

Reply:
Yes, of course!